pycbc/workflow/datafind.py

```diff
@@ -997,8 +977,14 @@ def run_datafind_instance(cp, outputDir, connection, observatory, frameType,
                          os.path.join(outputDir,'logs'), **dfKwargs)
     logging.debug("Asking datafind server for frames.")
     dfCache = lal.Cache.from_urls(
-        connection.find_frame_urls(observatory, frameType,
-                                   startTime, endTime, **dfKwargs),
+        find_frame_urls(
+            observatory,
+            frameType,
```

Comment:
PEP8 - snake_case please (though this is outwith this PR)

Reply:
Yeah, you can always tell old code written by me because of this. It does need changing, but there's a *lot* of it in workflow (and in tmpltbank), and that's a different PR to this one.